modules/ROOT/pages/change-jdk-config-in-projects.adoc

```diff
@@ -1,16 +1,16 @@
-= Change the JDK from Studio
+= Configure the JRE to Run the Embedded Mule Runtime Engine
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio 7.3 and later require JDK 1.8 to launch. These instructions change the JDK used to start up your Mule runtime in your Studio environment.
+Anypoint Studio 7.3 and later require JDK 1.8 to launch. These instructions change the JRE used to start up your Mule runtime in your Studio environment.
 
-You can select any of your installed JDKs as the default Java runtime environment that Studio uses to run your Mule project.
+You can select any of your installed JREs as the default Java runtime environment that Studio uses to run your Mule project.
 
-== Configuring the JDK Version for your Studio Projects
+== Configure the JRE Version for All your Studio Projects
```

Comment:
Again, isn't the user selecting a version of the JRE to use, not configuring a JRE?